src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala

```diff
@@ -175,7 +176,7 @@ trait FastSync {
 
     private var blockChainOnlyPeers = Seq.empty[Peer]
 
-    private val syncStatePersistCancellable = scheduler.schedule(persistStateSnapshotInterval, persistStateSnapshotInterval, self, PersistSyncState)
+    private val syncStatePersistCancellable = scheduler.schedule(0.seconds, persistStateSnapshotInterval, self, PersistSyncState)
```

Comment:
In this case this value shouldn't be configurable as while checking whether to allow starting regular sync or not, it is needed to [check whether fast sync had previously started](https://github.com/input-output-hk/etc-client/pull/287/files#diff-8c13c0426c84bb54f28f3662b267aa16R70).
For this it is checked whether any SyncState was saved, so right after `FastSync` starts, it's state should be saved to prevent it not being saved before `BestBlockNumber` is updated.